pkg/tfbridge/x/token_test.go

```diff
@@ -713,16 +713,18 @@ func TestMaxItemsOneAliasingNested(t *testing.T) {
                 "current": "pkg:index/r1:R1",
                 "fields": {
                     "f1": {
-                        "maxItemOne": false
+                        "maxItemsOne": false
                     },
                     "f2": {
-                        "maxItemOne": false,
-                        "fields": {
-                            "n1": {
-                                "maxItemOne": true
-                            },
-                            "n2": {
-                                "maxItemOne": false
+                        "maxItemsOne": false,
+                        "elem": {
```

Comment:
Oof. This is intentional?

Reply:
Yeah, for 2 reasons:
1. The property is actually called `MaxItemsOne`, so we may as well get it to match.
2. If anyone is using this in the wild (as in test PRs, this is still unreleased), this will prevent old (bad) data from infecting the next run.